src/shared/remoted_op.c

```diff
@@ -122,15 +125,17 @@ void parse_uname_string (char *uname,
                 // Get os_major
                 if (w_regexec("^([0-9]+)\\.*", *os_version, 2, match)) {
                     match_size = match[1].rm_eo - match[1].rm_so;
-                    *os_major = malloc(match_size +1);
-                    snprintf(*os_major, match_size + 1, "%.*s", match_size, *os_version + match[1].rm_so);
+
+                    if (*os_major = malloc(match_size +1), *os_major)
```

Comment:
Here we can do the same than @palaciosjeremias suggests about **os_malloc()**

Reply:
DONE